src/backends/simple.ts

```diff
@@ -0,0 +1,442 @@
+import type {
+    DecodedFrames,
+    Packet,
+    Demuxer,
+    Decoder,
+    Filterer
+} from '@antoinemopa/beamcoder';
+import beamcoder from '@antoinemopa/beamcoder';
+import type { ImageData } from 'canvas';
+import { createImageData } from 'canvas';
+import { BaseExtractor } from '../BaseExtractor';
+import type { Extractor, ExtractorArgs, InterpolateMode } from '../../framefusion';
+import { DownloadVideoURL } from '../DownloadVideoURL';
+
+const LOG = false;
+
+const createDecoder = ({
```

Comment:
This function is only called in _createDecoder and it's just one statement `return beamcoder.decoder()`, so I'd move it to `_createDecoder`

Reply:
Ok! Note that I think playbackRate might just need work on the calling code thought!

Reply:
You're probably right

Reply:
Unless we want to minterpolate as we discussed

Reply:
which I think we'll have to do when dealing with a low playbackRate